jekyll/_ccie/high-availability.md

```diff
@@ -0,0 +1,280 @@
+    ```
+5.	Create a circle-admin account to manage the databases as follows: 
+    ```
+    db.createUser({ user: 'circle-admin', pwd: '',
+                    roles: [{ role: 'root', db: 'admin' }, { role: 'dbAdmin', db: 'admin' }] })
+    ```
+    * Exit the client, uncomment the configuration, and restart.
+    * Reconnect to client as circle-admin and run the rs.initiate() command.
+	
+    **Note:** If you are using AWS and the first hostname resolves to something that is AWS-internal and your PRIMARY adopts that as its name, you must rename your host before extending the replica set by following the instructions [here](https://docs.mongodb.com/v3.2/tutorial/change-hostnames-in-a-replica-set/#change-all-hostnames-at-the-same-time).
+
+6. To add the remaining two hosts to the replica set, issue the following commands: 
+    ```
+    rs.add('<second hostname>')
+    rs.add('<third hostname>')
+    ```
+    The MongoDB replica set is ready for use with CircleCI.
+
+7. Create a CircleCI user for the actual application, which can be created with: 
+    ```
+    db.createUser({ user: "circle", pwd: '',
+                    roles: [ { role: "readWrite", db: "circle_ghe" },
+                             { role: "readWrite", db: "build_state_dev_ghe" },
+                             { role: "readWrite", db: "containers_dev_ghe" } ] })
+    ```
+
+### MongoDB Backups
+
+This section describes how to backup data mounted on EBS volumes.
+
+1. To ensure that the disk is in a consistent state, stop the mongodb process (using `sudo service mongod stop` or another system-appropriate command) on one of the SECONDARY instances and wait for the process to completely stop. **NOTE:** stopping the replica outright has proven to be more reliable for consistent restores than using the db.fsyncLock() mechanism described in the mongo documentation. This is also an additional safeguard for consistent state on top of the journal files.
+2. Use the AWS console or CLI to generate and complete a snapshot.
+3. To rejoin the replica set as a SECONDARY, restart the mongodb  process with `sudo service mongod start`.
+4. The SECONDARY begins serving traffic after replication catches up. 
+
+## Administering PostgreSQL
+
+It is assumed that users of the initial high-availability offering will use RDS on AWS, with multi-AZ mode and automated backups. Please let us know if any additional information is required for configuring and maintaining Postgres.
+
+## High Availability Configuration
+
+Making the services box automatically recover from failure means replacing it with an ASG of a single member, where the associated userdata entirely specifies how to install and configure Replicated and connect to the external databases.
+
+A working example of this can be found at [https://github.com/circleci/enterprise-setup/compare/ha-test](https://github.com/circleci/enterprise-setup/compare/ha-test). Note in particular the userdata for the services box launch configuration: 
```

Comment:
I think just replacing this with a link to the branch (rather than the diff) would be a bit nicer and good enough for now?